### PR TITLE
Update checkboxes for filtering for qualifications

### PR DESCRIPTION
**Pull request by tvararu**
### Screenshot (WIP)

screenshot 2018-10-01 at 12 31 37

### CODE REVIEW

**Comment by tvararu** [comment]
Checkboxes updated:

screenshot 2018-10-01 at 12 53 36

File: src/Filters/ResultsFilter.cs [hunk 1]
```diff
@@ -302,24 +306,21 @@ public ResultsFilter WithoutSubjects()
...
+            if (qualification.Any(x => x == QualificationOption.QtsOnly))
             {
-                yield return "PGCE (or PGDE) with QTS";
+                yield return "Qualified teacher status";
             }
-            if (true)
+            if (qualification.Any(x => x == QualificationOption.Other))
             {
-                yield return "Further Education (PGCE or PGDE without QTS)";
+                yield return "Other";
```
Comment (tvararu) [inline src/Filters/ResultsFilter.cs]: This should be `Further Education (PGCE or PGDE without QTS)` I believ, cc @fofr can you confirm?

File: src/Views/Filter/Qualification.cshtml [hunk 2]
```diff
@@ -61,12 +51,7 @@
               </div>
 
               <div class="govuk-checkboxes__item">
-                <input id="applyFilter-other" class="govuk-checkboxes__input" type="checkbox" name="other" value="true"
-                
-                @{ /* FIXME */ }
-                checked="true"
-                
-                >
+                <input id="applyFilter-other" class="govuk-checkboxes__input" type="checkbox" name="other" value="@QualificationOption.Other" @Model.qualification.Any(x => x == QualificationOption.Other)>
```
Comment (tvararu) [inline src/Views/Filter/Qualification.cshtml]: `checked=` is missing?

**Comment by tvararu** [comment]
Latest pass:

screenshot 2018-10-01 at 16 15 07

**Comment by tvararu** [comment]
Some variations of the filter on the left of the results page:

screenshot 2018-10-01 at 16 15 29

screenshot 2018-10-01 at 16 15 51
screenshot 2018-10-01 at 16 15 44
screenshot 2018-10-01 at 16 15 38

File: shared/ViewFormatters/CourseExtensions.cs [hunk 3]
```diff
@@ -27,6 +27,7 @@ public static string FormattedWebsite(this Course course)
 
         public static string FormattedOutcome(this Course course)
         {
+            // FIXME / REMOVE ME?
             if (course.IncludesPgce == IncludesPgce.No)
```
Comment (MartinBentley) [inline shared/ViewFormatters/CourseExtensions.cs]: This is not used now

**Pull request merged**